### PR TITLE
Provide example aktualizr configuration recipes and documentation

### PR DESCRIPTION
**Pull request by patrickvacek**
I have not actually entirely removed sota.env. It still exists to handle the things that can still only be done on the command line (i.e. --config and --secondary-config for virtual secondaries).

Note that this currently doesn't work until aktualizr properly handles config directories. You can at least test that the config gets put in the right place, though.

Test with `IMAGE_INSTALL_append =  " aktualizr-examples aktualizr-example-interface aktualizr-disable-send-ip aktualizr-log-debug"`.

### CODE REVIEW

File: recipes-sota/config/disable-send-ip.bb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+SUMMARY = "Disable IP reporting in Aktualizr"
```
Comment (lbonn) [inline recipes-sota/config/disable-send-ip.bb]: Should these recipes be prefixed with `aktualizr-`? Their names are not really helpful in isolation.

Comment (patrickvacek) [reply recipes-sota/config/disable-send-ip.bb]: I went back and forth on that. That probably makes sense.

Comment (patrickvacek) [reply recipes-sota/config/disable-send-ip.bb]: Done.

**Comment by patrickvacek** [comment]
I have updated this PR to now include all of the work relating to PRO-5308, including Serhiy's work to get the config working in systemd style. Please review since it would be nice to get this in before the end of the sprint.

File: recipes-sota/aktualizr/aktualizr-hsm-prov.bb [hunk 2]
```diff
@@ -18,15 +18,15 @@ require environment.inc
 require credentials.inc
 
 do_install() {
-    install -d ${D}${libdir}/sota
+    install -m 0700 -d ${D}${libdir}/sota
```
Comment (lbonn) [inline recipes-sota/aktualizr/aktualizr-hsm-prov.bb]: `sota/conf.d`?

Comment (patrickvacek) [reply recipes-sota/aktualizr/aktualizr-hsm-prov.bb]: Good call, fixed.

**Pull request merged**